.github/workflows/regression.yml

```diff
@@ -90,7 +92,7 @@ jobs:
         uses: docker://su2code/test-su2:20200303
         with:
           # -t <Tutorials-branch> -c <Testcases-branch>
-          args: -b ${{github.ref}} -t develop -c develop -s ${{matrix.testscript}}
+          args: -b ${{github.ref}} -t develop -c feature_species -s ${{matrix.testscript}}
```

Comment:
The testcases PR is also approved I think.

Reply:
Yep, before merging I will oc move the test to parallel regression and revert this

Reply:
d3b1914  species_regression.py ported